src/runtime/Makefile

```diff
@@ -173,7 +173,8 @@ DEFSHAREDFS := virtio-9p
 DEFSHAREDFS_QEMU_VIRTIOFS := virtio-fs
 DEFVIRTIOFSDAEMON := $(VIRTIOFSDBINDIR)/virtiofsd
 # Default DAX mapping cache size in MiB
-DEFVIRTIOFSCACHESIZE := 1024
+#if value is 0, DAX is not enabled
+DEFVIRTIOFSCACHESIZE := 0
```

Comment:
Makes sense to me.

Though, for 2.0, should we change the default to virtio-fs w/ out DAX for the 'qemu' configuration? I would think 2.0 is the time to switch from 9p -> virtiofs, unless there are stability concerns.

If we do the above, then I'd consider being able to enable DAX still for the 'qemu-virtiofs' configuration of Kata, but disabled for 'qemu' and 'clh' configurations...

Reply:
+1. We should open another issue to discuss it though. The PR meant to forward port a 1.x commit.